shared/src/data/default-state/material-templates.ts

```diff
@@ -0,0 +1,34 @@
+import { CanCaterFor } from '../../models/utils';
+import { MaterialTemplate } from '../../models/material-template';
+import type { MaterialType } from '../../models/utils/material-type';
+
+const defaultMaterialTemplate = MaterialTemplate.create(
+    'default',
+    {
+        url: '/assets/material.svg',
+        height: 40,
+        aspectRatio: 1,
+    },
+    CanCaterFor.create(2, 0, 0, 'and'),
+    2.5,
+    5.5
+);
+
+const bigMaterialTemplate = MaterialTemplate.create(
+    'big',
+    {
+        url: '/assets/material.svg',
+        height: 80,
+        aspectRatio: 1,
+    },
+    CanCaterFor.create(2, 2, 0, 'and'),
+    15,
+    0
```

Comment:
If we include the big material, the overwriteRange should be the same or more similar to the default material.
```suggestion
    2.5,
    15
```